File: Ceros de una matriz/A3.py
NumerosMatriz = "1 0 3 0 2 3"
x = "-1"
d = "2 3"
Lista = NumerosMatriz.split()
Dimensiones = d.split()

Matriz = []

for i in range(0, int(Dimensiones[0])):
	Matriz.append([])
	
	for j in range(0, int(Dimensiones[1])):
		Matriz[i].append(Lista[0])
		del Lista[0]

resp = ""
c = 0
fila = 0

for i in Matriz:
	fila += 1
	columna = 0
	for j in i:
		columna += 1 
		if j == "0":
			c += 1
		
			resp = resp + "(" + str(fila) +"," + str(columna) +  ") "

print(str(c) + " " + resp)

File: Ceros de una matriz/A3.js
const NumerosMatriz = "1 0 3 0 2 3"
const x = "-1"
const d = "2 3"
const Lista = NumerosMatriz.split(' ')
const Dimensiones = d.split(' ')

let Matriz = []

for(let i = 0; i < Dimensiones[0]; i++){
    Matriz.push([])
    for(let j = 0; j < Dimensiones[1]; j++){
        Matriz[i].push(Lista[0])
		Lista.shift();
    }
}


let resp = ""
let c = 0
let fila = 0

for(let i = 0; i < Matriz.length; i++){
    fila += 1
	let columna = 0
    
    for(let j = 0; j < Matriz[i].length; j++){
        columna += 1
		if (Matriz[i][j] == "0"){
            c += 1
            resp = resp + "(" + fila +"," + columna +  ") "
        }

    }

}
console.log(`${c} ${resp}`)
